ibis/backends/base/sql/registry/window.py

```diff
@@ -254,9 +254,9 @@ def window(translator, expr):
     )
 
     _unsupported_reductions = (
-        ops.CMSMedian,
+        ops.ApproxMedian,
```

Comment:
We do not need to have them both here, because this list is only used by an `isinstance` which handles class inheritance. It will check if a class is derived from a base class in the tuple.